Dockerfile

```diff
@@ -1,13 +1,26 @@
-FROM python:3.11.3
+FROM python:3.11
+
+# set environment vars
 ENV PYTHONUNBUFFERED True
+ENV APP_HOME /root
```

Comment:
maybe use something outside of root like /opt/ as ideally one does not want to run the container as the root user. 

Not sure if we need to worry about adding an other user in this docker image though but at least good to make it possible.